client/app/config.go

```diff
@@ -195,6 +195,9 @@ func (cfg *Config) Web(c *core.Core, mm *mm.MarketMaker, log dex.Logger, utc boo
 		UTC:           utc,
 		CertFile:      certFile,
 		KeyFile:       keyFile,
+		NoEmbed:       cfg.NoEmbedSite,
+		HttpProf:      cfg.HTTPProfile,
```

Comment:
Sorry one more thing. Did you intent to add these two? It actually looks like they should be there...